src/main/groovy/net/anshulverma/gradle/estilo/checkstyle/checks/CheckType.groovy

```diff
@@ -18,14 +18,24 @@ package net.anshulverma.gradle.estilo.checkstyle.checks
 /**
  * @author Anshul Verma (anshul.verma86@gmail.com)
  */
-public enum CheckType {
+public class CheckType {
 
-  GOOGLE('google_checks.xml'),
-  SUN('sun_checks.xml')
+  public static GOOGLE = new CheckType('google_checks.xml')
+  public static SUN = new CheckType('sun_checks.xml')
+  public static EMPTY = new CheckType('empty_checks.xml')
+  public static CUSTOM = new CheckType("config/checkstyle.xml")
```

Comment:
I tried to get the `enum`-like behavior by representing the previously-enumerated values as public static fields.

Reply:
It would be best if we can keep these as a enumerations. 
How about this:
- Create a function that returns the path of checkstyle file
- In that function see if the path matches any of the names of the enums (just like in `valueOf`)
- If it matches then use the file name in the enum as `"/${filename}"` for the full path
- If it does not match you can simply use the name arg as the path